docs/teaching.rst

```diff
@@ -0,0 +1,85 @@
+.. _teach:
+
+Teaching with the DataLad Handbook
+----------------------------------
+
+The handbook is a free and open source educational instrument made available
+under a Creative Commons Attribution-ShareAlike (CC-BY-SA) license [#f1]_.
+We are happy if the handbook serves as a helpful tool for other trainers, and
+try to provide many useful additional teaching-related functions and contents.
+Below, you can find them listed:
+
+Use the handbook as a textbook/syllabus
+"""""""""""""""""""""""""""""""""""""""
+
+The Basics sections of the handbook is a stand-alone course that you can refer
+trainees to. Regardless of background, users should be able to work through this
+part of the book on their own. From our own teaching experiences, it is feasible
+and useful to work through any individual basics chapter in one go, and assign
+them as weekly or bi-weekly readings.
+
+Use slides from the DataLad course
+""""""""""""""""""""""""""""""""""
+
+In parallel to the handbook, we are conducting data management workshops with
+attendees of every career stage (MSc students up to PIs). The sessions are either
+part of a lecture series (with bi-weekly 90 minute sessions) or workshops of different
+lengths. Sessions in the lecture series are based on each chapter. Longer workshops
+combine several chapters. You can find the slides for the workshops in the
+`companion course repository <https://github.com/datalad-handbook/course>`_.
+Slides are made using `reveal.js <https://github.com/hakimel/reveal.js/>`_.
+They are available as PDFs in ``talks/PDFs/``, or as the source ``html`` files
+in ``talks/``.
+
+Enhance talks and workshops with code casts
```

Comment:
I still don't think that 'cast' is a term that can be intuitively understood. I think "demo" is a better choice.